styles.css

```diff
@@ -9,6 +9,8 @@
 
 body {
 	background: rgb(35, 69, 130);
+	overflow-x: hidden;
```

Comment:
Setting the `overflow` of the content on the body to `hidden` isn't considered to be best practice. This could be mitigated by setting the unit of the `width` of both the `header` and `footer` class to `%`, which aligns the width in respect to its parent container, instead of `vw`.